src/main/java/org/mockbukkit/mockbukkit/entity/LivingEntityMock.java

```diff
@@ -652,7 +652,15 @@ public EntityPotionEffectEvent addPotionEffect(@NotNull PotionEffect effect, Ent
 
 		if (!event.isCancelled())
 		{
-			activeEffects.computeIfAbsent(type, k -> new PriorityQueue<>()).add(new ActivePotionEffect(effect));
+			var pq = activeEffects.computeIfAbsent(type, k -> new PriorityQueue<>());
```

Comment:
Var 😭 . Haha, just me being a stubborn man refusing to use that keyword

Reply:
It makes coding sooooo much easier and cleaner!